heron/stmgr/src/cpp/manager/instance-server.cpp

```diff
@@ -154,8 +157,23 @@ InstanceServer::~InstanceServer() {
     }
   }
 
+  for (auto qlmIter = connection_buffer_length_metric_map_.begin();
```

Comment:
a little complicated, please add comment.

Reply:
comment added and simplified in next commit.